plugins/agent_types/AcquiaLiftTarget.inc

```diff
@@ -61,4 +61,20 @@ class AcquiaLiftTarget extends PersonalizeAgentBase implements PersonalizeExplic
   public function supportsMultipleDecisionPoints() {
     return FALSE;
   }
+
+  /**
+   * Implements PersonalizeAgentGoalInterface::useClientSideGoalDelivery().
+   *
+   * @return boolean
+   */
+  public function useClientSideGoalDelivery() {
+    return TRUE;
+  }
+
+  /**
+   * Implements PersonalizeAgentGoalInterface::sendGoal().
+   */
+  public function sendGoal($goal_name, $value = NULL) {
+    return;
+  }
```

Comment:
Well - I didn't actually do anything with it the implementation.  I only implemented it because I was using it to determine if we should show goals.  That's how we used it in unibar too (and possibly some other places).